framework/src/main/java/org/tron/core/net/peer/PeerConnection.java

```diff
@@ -225,7 +225,8 @@ public String log() {
         channel.getInetSocketAddress(),
         (now - channel.getStartTime()) / Constant.ONE_THOUSAND,
         channel.getAvgLatency(),
-        fastForwardBlock != null ? fastForwardBlock.getNum() : blockBothHave.getNum(),
+        fastForwardBlock != null ? fastForwardBlock.getNum() : String.format("%d [%ds]",
```

Comment:
Should it be "%d [%dms]"?

Reply:
It means how many seconds ago, not milliseconds. (System.currentTimeMillis() - blockBothHaveUpdateTime) / 1000).